english/wikipedia/wiki.md

```diff
@@ -14,6 +14,8 @@ And you don't have to go to the library.
 
 [Behold!](http://en.wikipedia.org)
 
+You may also like it's sister site, wikiyoyage (https://www.wikivoyage.org) with more detailed information on travel destinations.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.